magpie/owsrequest.py

```diff
@@ -26,31 +26,29 @@ def ows_parser_factory(request):
     Default to JSON if no ``Content-Type`` is specified or if it is 'text/plain' but can be parsed as JSON. Otherwise,
     use the GET/POST WPS parsers.
     """
```

Comment:
adjust docstring to reflect change (ie: attempt parsing JSON if body is also available)